appveyor.yml

```diff
@@ -7,7 +7,7 @@ install:
   # Get the latest stable version of Node.js or io.js
   - ps: Install-Product node $env:nodejs_version
   # install modules
-  - npm install -g cordova@7.0.1 ionic@3.4.0
+  - npm install -g cordova@7.0.1 ionic@3.12.0
   - npm install
```

Comment:
Should be aligned with the latest version in the Dockerfile? I don't think that's 3.12

Reply:
3.9.2 is the correct one